app/api-react.js

```diff
@@ -1,36 +1,46 @@
 import buildStubbedApiReact from './utils/api-react'
-import { TYPE_MARKDOWN, TYPE_VOTING } from './utils/constants'
+import {
+  TYPE_MARKDOWN,
+  TYPE_VOTING,
+  VOTING_STATUS_ONGOING,
+  VOTING_STATUS_PENDING_ENACTMENT,
+} from './utils/constants'
 import { ipfs } from './utils/ipfs'
+
 const initialState = process.env.NODE_ENV !== 'production' && {
-  // widgets: [
-  //   {
-  //     type: TYPE_VOTING,
-  //     data: {
-  //       votes: [
-  //         {
-  //           status: 'Passed',
-  //           id: 1,
-  //           description: 'Mint 1 ETH for 0xbaB65A7D8a8F67D38cD26af113b95279fA3E7616',
-  //           percentageYes: 0.6134,
-  //         },
-  //         {
-  //           status: 'Rejected',
-  //           id: 2,
-  //           description: 'Mint 19 ETH for 0x978f4De6B4833e956a366A441B277deB7A7C0d4c',
-  //           percentageYes: 0.3196,
-  //         },
-  //       ],
-  //     },
-  //     position: {},
-  //   },
-  //   {
-  //     type: TYPE_MARKDOWN,
-  //     data: {
-  //       content: '# hola, `mundo`!',
-  //     },
-  //     position: {},
-  //   },
-  // ],
+  widgets: [
+    {
+      type: TYPE_MARKDOWN,
+      data: {
+        content: '# hola, `mundo`!',
+      },
+      position: {},
+    },
+    {
+      type: TYPE_VOTING,
+      data: {
+        votes: [
```

Comment:
I am still not sure which is the best place for the `votes` array, just exploring it to be in the root state, the benefits are related to building it from the `store` are better consuming with `useAragonApi` instead of passing by props, also we avoid redundancy in the remote case multiple `Votes` widgets are added

Reply:
Solved